app/models/project.rb

```diff
@@ -36,4 +36,49 @@ def at_least_2_interests
       false
     end
   end
+
+  def recommended_users
+    ideal_user = User.create(username: "ideal user",
+    about_me: "This is an ideal user generated by the project's skills and
+    interests and temporarily used for the purpose of finding releated
+    users",
+    first_name: "ideal",
+    last_name: "user",
+    zip_code: "02445",
+    skill_list: self.skill_list.join(", "),
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Redundant `self` detected.